pkg/template/api/v1/types.go

```diff
@@ -14,6 +14,9 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
+	// Optional: An instructional message for the user processing this template (escaped parameters will be substitued).
```

Comment:
i'm a little fuzzy on the api doc rules these days but i think this needs to start with either:
// postProcessMessage blah blah
or
// Optional: postProcessMessage blah blah

and based on how we're doing it in most places, i don't think we want // Optional (feel free to fix the existing instances if i'm correct).

@smarterclayton ?